### ISSUE TITLE
11/10/2021: Refactor the code architecture.

### ISSUE THREAD

**Issue by angemaggy** [Open-Science-Community-Saudi-Arabia/OSCSA_Website#120]
The actual site structure is really bad and the code is not reusable. I suggest to use `php` or another backend language of your choice to make some reusable layouts(templates) in order to well organised the architecture of the code. I could easily integrate the translation file with multiple languages (en, arabic, ...) to avoid using a new file each time we want to implement a page.
![current_architecture](https://user-images.githubusercontent.com/100047512/195105429-ab745003-683b-493d-b9ea-1037919faaeb.png)
In the new architecture, we will have one index file, a file for translation and the orthers pages using a common layout, header and footer.
To summarize:

- restructure the code architecture;
- make reusable codes (layout...);
- better way to translate our files without duplicate pages;
- propose to work with `php` to make the process easy or another language of your choice(any one);

I will like to work on some or all of those points depending on which one you will think are necessary.

**Comment by angemaggy** [comment]
@BatoolMM what do you think about it ?
